manifests/module.pp

```diff
@@ -71,14 +75,20 @@
     default => $module_prefix,
   }
 
+  $real_install_options = $install_options ? {
+    ''      => $php::install_options,
+    default => [],
```

Comment:
But then `$real_install_options` would be an empty string.
It needs to be an empty array.